### ISSUE TITLE
Auto run when starting & memorise script

### ISSUE THREAD

**Issue by Fotia13** [Topaz-Reality/LuaFrontend#1]
Will it be possible to add an option to auto run the engine when starting LuaFrontend ?
Like with a command or in a config file or something else, because I have made a batch for LuaBackend that is like this: 
```
@echo off
::Start KH2 game
start "" "C:\Program Files\Epic Games\Kingdom Hearts HD 1 5 + 2 5 ReMIX\KINGDOM HEARTS II FINAL MIX.exe"

::Wait 5 seconds for the game starts
timeout 5

::Start LuaFrontend app
start "LuaFrontend" "C:\Users\user\Desktop\Gaming Room\LuaFrontend\LuaFrontend.exe"

::Chang the default audio driver for KH2
SetAppDefault\SoundVolumeView.exe /SetAppDefault "VB-Audio Hi-Fi Cable\Device\Hi-Fi Cable Input\Render" 0 "KINGDOM HEARTS II FINAL MIX.exe"
```

and I wanted to switch to LuaFrontend.
I know it is a little thing.
Also an other little thing.
Can it be possible to memorise the scripts I had chose when restarting the applications ?

**Comment by Topaz-Reality** [comment]
Option memorization is in the works. However, because of the way that scripts are added and removed, script selection memorization will not be implemented.

As for the auto-start, that will be taken into consideration.

**Comment by Fotia13** [comment]
Thanks a lot

**Comment by Topaz-Reality** [comment]
Option memorization has been implemented as of v1.10.
Auto-start will not be implemented, but it may be in the distant future.
Closing.
